### PR TITLE
Update scripts to python3

### PR DESCRIPTION
**Pull request by henrik-dmg**
Since python2 reached EOL, all scripts should be updated to use python3 syntax.
Changes made:

- Initial conversion of all python scripts using `2to3 -w scriptname.py`
- Executed all scripts multiple times to find possible errors that weren't caught by the conversion scripts
- most of the changes are related to braces in print statements

### CODE REVIEW

File: attributions2json.py [hunk 1]
```diff
@@ -65,32 +66,33 @@ def get_license_text(repo):
     path = find_license_path(repo)
     if path:
         data = send_content_request(repo, path)
-        return data['content'].decode(data['encoding'])
+        decodedData = base64.b64decode(data['content'])
```
Comment (Johennes) [inline attributions2json.py:72]: Should we still be reading `data['encoding']` or at least assert that it is `"base64"`?

Comment (henrik-dmg) [reply attributions2json.py:72]: Yeah good idea, how about this?
```suggestion
if data['encoding'] != 'base64':
    print("Encoding of Github response was not base64")
    sys.exit(1)
decodedData = base64.b64decode(data['content'])
return decodedData.decode('utf-8')
```

Comment (Johennes) [reply attributions2json.py:72]: Seems good enough for now since we haven't seen any other encoding so far.

**Review by henrik-dmg — COMMENTED** [review]
Implemented discussed assertion of non-base64 encoding

**Comment by Johennes** [comment]
Seems good to merge!

**Pull request merged**